### ISSUE TITLE
Custom Chooser view example.

### ISSUE THREAD

**Issue by csfelipe** [linkedin/Hakawai#46]
Hi.

I'm trying to replicate "the marking someone in a post comment" situation. When you have a keyboard up and there's a tableview that rises with the possible entities to mark. (Image reference: [http://imgur.com/a/w58jK](http://imgur.com/a/w58jK))

Any insights are welcomed. The documentation is very scarce. 

Thanks!

**Comment by almas73** [comment]
@csfelipe you can initialize HKWMentionsPlugin with `customLockBottomNoArrow` chooser mode, and then configure the callback where you can place chooser view anywhere you want:
```
mentionsPlugin.setChooserTopLevel(messageAccessoryView.mentionsContainer) {  chooserView in
    // add `chooserView` as a subview
}
```

**Comment by yanyuanjie** [comment]
@almas73 Custom views can be added. But how to add tags to this TextView?

**Comment by JaniPurvi** [comment]
I’m still unable to make exact look and feel just like linkedIn @mention view. Even though added custom view it lies inside the textview but I want to set textview size 50 in commentbox so whenever I decrease textview size it decreases chooserview automatically. I have tried lot many soutions like disabling constraints,set frame of chooserview, removed parent view , set chooser top level,adjust mode without arrow and much more still I’m not able to achieve http://imgur.com/a/w58jK
If anybody has achieved please share the code.

**Comment by dgattey** [comment]
Yup, our mentions view has a lot of LinkedIn specific styling and behavior on top of it. What exactly can we help clarify here? Is there something specific you're trying to do from that view?

**Comment by dineshiOSDev** [comment]
> @csfelipe you can initialize `HKWMentionsPlugin` with `customLockBottomNoArrow` chooser mode, and then configure the callback where you can place chooser view anywhere you want:
> 
> ```
> mentionsPlugin.setChooserTopLevel(messageAccessoryView.mentionsContainer) {  chooserView in
>     // setup `chooserView` constraints
> }
> ```

Can you please share some example code to set the custom frames for chooser view?
